### ISSUE TITLE
Missing new lines on diagnostics commands results

### ISSUE THREAD

**Issue by kedare** [netfishers-onl/Netshot#112]
Hello,

I have been giving a try to the diagnostics commands,
The command is properly executed, but it looks like the new lines are not shown properly in the web interface

![image](https://user-images.githubusercontent.com/359877/113405521-436ef280-93aa-11eb-8a6a-0ff357cca18b.png)

On the web API is shows with hardcoded `\r\n` (I suppose this one is expected?)

```json
[
  {
    "type": "TEXT",
    "creationDate": 1616602149570,
    "lastCheckDate": 1616602149570,
    "text": "  \r\n  \r\n  \r\nFan:\r\n---------------------------------------------------------------------------\r\nFan             Model                Hw     Direction       Status\r\n---------------------------------------------------------------------------\r\nFan1(sys_fan1)  NXA-FAN-160CFM-PI    --     front-to-back   Ok \r\nFan2(sys_fan2)  NXA-FAN-160CFM-PI    --     front-to-back   Ok \r\nFan3(sys_fan3)  NXA-FAN-160CFM-PI    --     front-to-back   Ok \r\nFan_in_PS1                           --     front-to-back   Ok\r\nFan_in_PS2                           --     front-to-back   Ok\r\nFan Zone Speed: Zone 1: 0x66\r\nFan Air Filter : NotSupported\r\n\r",
    "diagnosticName": "Fan monitoring"
  }
]
```

Putting in in a `<pre></pre>` on the web UI should do it.

Thanks

**Comment by kedare** [comment]
I tried to do a PR for this but I could not get the change to show up after compiling

This is my `deviceDiagnostic.html` file after the update:

```
<tr>
		<td class="maincolumn"><%- diagnosticName %></td>
		<td>
			<pre>
				<% if (typeof assumption !== "undefined") { %>
					<%- (assumption ? "TRUE" : "FALSE") %>
				<% } else if (typeof number !== "undefined") { %>
					<%- number %>
				<% } else if (typeof text !== "undefined") { %>
					<%- text %>
				<% } %>
			</pre>
		</td>
		<td><%- formatDateTime(creationDate) %></td>
		<td><%- formatDateTime(lastCheckDate) %></td>
	</tr>
```

Then `mvn clean; mvn package` but no visible change, am I missing something ?

**Comment by SCadilhac** [comment]
Apart from browser cache, I don't see what was wrong.
I just implemented the fix, see f72ce417e8dda37acdd8ddb5bdab21eb162f86b4.
Thanks!
